contracts/src/v0.6/Flags.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.6.6;
```

Comment:
Can we add [typeAndVersion](https://github.com/smartcontractkit/chainlink/blob/arbitrum-flag-validator/contracts/src/v0.7/interfaces/TypeAndVersionInterface.sol) to this? Probably "Flags 2.0.0".